apis/k8ssandra/v1alpha1/k8ssandracluster_types.go

```diff
@@ -61,6 +61,12 @@ type K8ssandraClusterSpec struct {
 	// If this is non-nil, Medusa will be deployed in every Cassandra pod in this K8ssandraCluster.
 	// +optional
 	Medusa *medusaapi.MedusaClusterTemplate `json:"medusa,omitempty"`
+
+	// During a migration or in hybrid mode the operator should alter keyspaces replication settings including these external DCs.
```

Comment:
We chatted about this a little this afternoon... let's tweak the comment just a bit to remove the bits about hybrid mode.  I think the hybrid piece is really interesting, for a number of reasons, but for now until we think through what it looks like to run a hybrid cluster long term let's only state that we're doing this for temporary migration purposes.  Just trying to pull in our potential scope in terms of documentation/questions/support for now.

Reply:
kk, I made the changes 👍